tasks/testnet.js

```diff
@@ -18,6 +20,13 @@ async function main() {
     }
     const TESTNET_NODE_FOLDER = path.join(userHome, `.gaiad-testnet`)
     startLocalNode(TESTNET_NODE_FOLDER)
+    if (process.env[`VALIDATORS`] && process.env[`VALIDATORS`] > 1) {
+      // let's re-t
```

Comment:
?

Reply:
Ah this should have been stashed sorry.
Basically I planned to use the cli() feature to specify how many nodes you want + done some experiments to add accounts.

I moved here the idea about extra parameters that I was playing experimenting on:
https://github.com/cosmos/voyager/issues/1752

Reply:
:D no problem. I like the proposal.